src/managed/Microsoft.Extensions.DependencyModel/DependencyContextWriter.Utf8JsonWriter.cs

```diff
@@ -22,7 +23,11 @@ public void Write(DependencyContext context, Stream stream)
             }
             using (var bufferWriter = new ArrayBufferWriter())
             {
-                var options = new JsonWriterOptions { Indented = true };
+                // Custom encoder is required to fix https://github.com/dotnet/core-setup/issues/7137
+                // Since the JSON is only written to a file that is read by the SDK (and not transmitted over the wire),
+                // it is safe to skip escaping certain characters in this scenario 
+                // (that would otherwise be escaped, by default, as part of defense-in-depth, such as +).
+                var options = new JsonWriterOptions { Indented = true, Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping };
```

Comment:
Is this any more unsafe than what Json.Net did before?

Reply:
Nope.